### ISSUE TITLE
Throwing a trident has wrong angle shown

### ISSUE THREAD

**Issue by bundabrg** [GeyserMC/Geyser#1935]
**Describe the bug**

When throwing a trident it appears sideways to the player throwing it

**To Reproduce**

1. Throw trident

**Screenshots / Videos**

![](https://media.giphy.com/media/AkKbtWYhG1tFCorCSc/giphy.gif)

**Server Version and Plugins**

If you're running a multi-server instance, or using Geyser Standalone:

Paper:
```
[15:40:05 INFO]: This server is running Paper version git-Paper-204 (MC: 1.16.3) (Implementing API version 1.16.3-R0.1-SNAPSHOT)
[15:40:05 INFO]: Checking version, please wait...
[15:40:07 INFO]: Previous version: git-Paper-38 (MC: 1.16.1)
[15:40:07 INFO]: You are 49 version(s) behind
```

**Geyser Dump**

https://dump.geysermc.org/vYBp6L0wkocOibNVwUCTcnJBq4h0Wjwb

**Minecraft: Bedrock Edition Version**

1.16.201.1, linux apk launcher

**Comment by MacacoLew** [comment]
I can confirm that

**Comment by Camotoy** [comment]
I have also seen this error present with arrows.

**Comment by Camotoy** [comment]
Fixed with #1929.
